### DISCUSSION TITLE
Bots getting stuck on some ladders in submarines

### DISCUSSION THREAD

**Discussion by Exoteklas** [FakeFishGames/Barotrauma#13600] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

On 2 occasions in submarines I have experienced that there are places where bots get trapped in stairs trying to go to a task, repair, fill air tanks, etc. It has happened to me with the submarine "Camel" and the "typhon 1" simply The characters are trapped there without being able to do anything until I control them manually, this can be a bit annoying when playing alone since, for example, if the engine in "typhon" is damaged, the bots cannot reach it and I have to move them to get there let them fix it, I will attach a photo of where they are stuck and also a video as proof, also a photo of the version of the game and just to be clear, the game is not pirated, it is purchased on steam

https://github.com/FakeFishGames/Barotrauma/assets/163709678/5f3ae06d-e69c-430d-88a8-ef94d8f11f60

![image](https://github.com/FakeFishGames/Barotrauma/assets/163709678/321029f9-d2a7-4525-8dbe-1d1f7949f7a0)
![VERSION](https://github.com/FakeFishGames/Barotrauma/assets/163709678/0a2e3d26-b753-4041-9494-3539fdbdade4)

### Reproduction steps

just add as a task, repair cracks, engine or fill oxygen cans in typhon

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by pavel4win** [comment]
I had bots do the same thing on the Winterhalter sub

**Comment by itchyOwl** [comment]
Converted to an issue ticket: #13744

**Comment by itchyOwl** [comment]
Wasn't able to repro this in Typhon. Tried to make a bot follow me, clean up items (dropped some items around), fix leaks (caused some leaks), and the bot was able to path through the ladders just ok. Needs better steps to repro, moving to QA investigates.

If you can repro the issue, type debugai. Makes it easier to see what's going on there.

**Comment by NotWendy** [comment]
Potentially related to bots being unable to judge the size of the items' interaction areas while they're on a ladder. Detailed here:
https://github.com/FakeFishGames/Barotrauma/discussions/13598
Commented under the issue ticket as well.

**Comment by Regalis11** [comment]
Thank you for the report! The issue has been addressed in our private repo now, fix coming up in a future unstable build.
